engine/schema/pom.xml

```diff
@@ -122,7 +122,8 @@
                             <goal>wget</goal>
                         </goals>
                         <configuration>
-                            <url>https://download.cloudstack.org/systemvm/${cs.version}/md5sum.txt</url>
+<!--                            <url>https://download.cloudstack.org/systemvm/${cs.version}/md5sum.txt</url>-->
```

Comment:
remove comment code that isn't used

Reply:
I've temporarily created a directory to place the 4.17 systemvm template for testing ,  which post testing will be removed. This was done so as to ensure that travis doesn't fail

Reply:
Just a reminder on this, to be removed when ready